### PR TITLE
Revert "Accesibility changes"

### PR DESCRIPTION
**Pull request by Javivy**
# In this milestone we:

## Make page accessible for everyone

  - Change the  'About me' section header for a better flow of the page
  - Make elements not overlap with other elements
  - Link the navbar links to their respective section
  - Work with a coding partner
  - Link accounts to the social media links

## Checks ✔️ 

  - Page Titles - Correct ✅ 
  - Image text alternatives - Correct ✅ 
  - Text Headings - Line 163 of HTML file
  - Color contrast - Correct ✅ 
  - Resize - Correct ✅ 
  - Interaction - Correct ✅ 
  - Moving content - Correct ✅ 
  - Multimedia - Correct ✅ 
  - The basic structure of the page - Correct ✅

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,32 +50,32 @@ <h1>
...
+          <a href="#"><img src="icons/chevron-down.png" alt="Icon" /></a>
         </div>
       </div>
     </header>
 
     <main>
       <!-- RECENT WORKS SECTION -->
 
-      <section id="recent-work" class="works-section">
+      <section class="works-section">
```
Comment (vickymarz) [inline index.html:78]: - [ ] Kindly ensure that your webpage has a minimum contrast ratio of at least 4.5:1.  You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project 👍🏽 .

Here is a failing result of the contrast check. Kindly fix it 👍🏽 

image

image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @Javivy , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅  
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [ ] For easy accessibility of your website, I think it will be better if your project title adequately and briefly describes the content of the page e.g using your name (i.e **"Javivy Portfolio"**) as the title is much better 😄 .
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Javivy) [reply index.html:78]: @vickymarz Hi, shouldn't I leave it like that since the design uses those colors?

Comment (OscarFM-dev) [reply index.html:78]: Hello @vickymarz thank you for reviewing our work! We had that discussion during our review, should we change the colors from the design to match the accessibility goals or should we stick to the design template from Figma? Please let us know to work towards the purpose of the project!

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the accessibility ✅
📌 Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (vickymarz) [reply index.html:78]: @OscarFM-dev as regards the button, your hover color and your default color looks the same. you should implement the hover state to at least improve accessibility

**Pull request merged**